spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -229,5 +229,43 @@
         end
       end
     end
+
+    describe "household rent own status" do
+      context "when taxpayer is a renter" do
+        let(:intake) { create(:state_file_nj_intake, :df_data_minimal, household_rent_own: 'rent',) }
+
+        it "adds a checked tenant element to property tax deduct or credit" do
+          expect(xml.at("PropertyTaxDeductOrCredit Tenant").text).to eq("X")
+        end
+
+        it "does not add a checked homeowner element to property tax deduct or credit" do
+          expect(xml.at("PropertyTaxDeductOrCredit Homeowner")).to eq(nil)
+        end
```

Comment:
[dust] if you want, it might be easier / more readable to combine these into a single it statement with 2 expect statements (here and below)